CONTRIBUTING.md

```diff
@@ -0,0 +1,224 @@
+# Contributing
+
+For the development of @ebury/chameleon-components, we use storybook [Storybook](https://storybook.js.org/).
+If you want to contribute to the library then you must do the following to set up your local environment:
+
+## Storybook
+
+1\. Clone chameleon-components repo
```

Comment:
this is the instalation guide and goes in the readme https://bitbucket.org/fxsolutions/eburyonline/src/master/README.md

Reply:
Here there are two things, the installation to develop/contribute on the library or the installation to use the library in other projects, I guess the one to develop should be on the contributing and the other one on the README, and we already have these two installation process on the files.